.github/workflows/release-check.yml

```
@@ -0,0 +1,77 @@
+on:
+  pull_request:
+    branches:
+      - prod
+
+jobs:
+  check_branch:
+    name: Verify Branch Source
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check Source Branch
+        run: |
```

actionlint (1.7.8)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)